app/src/main/java/br/org/cesar/knot_setup_app/fragment/configureGatewayWifi/ConfigureGatewayWifiPresenter.java

```diff
@@ -1,16 +1,16 @@
-package br.org.cesar.knot_setup_app.activity.configureGatewayWifi;
+package br.org.cesar.knot_setup_app.fragment.configureGatewayWifi;
+
 
 import br.org.cesar.knot_setup_app.KnotSetupApplication;
-import br.org.cesar.knot_setup_app.activity.configureGatewayWifi.ConfigureGatewayWifiContract.ViewModel;
-import br.org.cesar.knot_setup_app.activity.configureGatewayWifi.ConfigureGatewayWifiContract.Presenter;
 import br.org.cesar.knot_setup_app.domain.callback.DeviceCallback;
+import br.org.cesar.knot_setup_app.fragment.configureGatewayWifi.ConfigureGatewayWifiContract.ViewModel;
+import br.org.cesar.knot_setup_app.fragment.configureGatewayWifi.ConfigureGatewayWifiContract.Presenter;
 import br.org.cesar.knot_setup_app.model.BluetoothDevice;
 import br.org.cesar.knot_setup_app.utils.Constants;
 import br.org.cesar.knot_setup_app.wrapper.BluetoothWrapper;
 import br.org.cesar.knot_setup_app.wrapper.NetworkWrapper;
 
-public class ConfigureGatewayWifiPresenter  implements Presenter {
-
+public class ConfigureGatewayWifiPresenter implements Presenter{
```

Comment:
Why change this line?
You are fixing one coding style issue (double spacing) and adding another (no space before `{`).
As discussed, I'd suggest fix all this coding style issues in a final commit in this patchset, since they are already in the mainline.

Reply:
Done.

Reply:
Not done. You are still changing the code style in an incorrect patch.